docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.7'
+
+services:
+  opencollective_api:
+    build:
+      context: . # Build the api image from the Dockerfile
+    command: ${OPENCOLLECTIVE_DOCKER_INIT_CMD} # Use this env var to send pre-defined and custom commands
+    ports:
+      - '3060:3060' # App port
+      - '1080:1080' # Mail gui port
+      - '1025:1025'
+      - '9229:9229' # Debug port
+    volumes:
+      - .:/usr/src/app # Mount the app in the container
+    depends_on:
+      - opencollective_db # Add dependency on the db service
+    environment:
+      - PGHOST=opencollective_db # The db host is the name of the service
+      - PGUSER=postgres
+      - PG_URL=postgres://opencollective@opencollective_db:5432/opencollective_dvl
+      - PG_MAINTENANCE_URL=postgres://opencollective_db:5432/postgres
+
+  opencollective_db:
+    image: mdillon/postgis:9.6
+    volumes:
+      - opencollective-db-data:/var/lib/postgresql/data # Persist data between mounts
```

Comment:
Is that going to work on Windows?

Reply:
It should. In this case a named volume is created with the name _opencollective-db-data_ associated to the location _/var/lib/postgresql/data_ in the container. You can check this name with the command `docker volume ls`.

Here is an overview on the [different kinds of volumes](https://success.docker.com/article/different-types-of-volumes). We could also remove this section and create an anonymous volume but as the doc says it can be difficult to refer to the same volume over time.